rest/data/interface.go

```diff
@@ -229,4 +230,5 @@ type Connector interface {
 	// Commit queue methods
 	GithubPREnqueueItem(string, string, int) error
 	EnqueueItem(string, string, string, string) error
+	FindCommitQueueByID(string) (*commitqueue.CommitQueue, error)
```

Comment:
I know we've been wildly inconsistent with this on the past but I've started to think that the connector should really not have interface dependencies on the database models and just return rest models. 

Would it be too much of a bother to change that here?

Reply:
done.